### PR TITLE
OT147-63 Added social network URL to Organization

### PR DESCRIPTION
**Pull request by pablofrankenn**
### **Summary:**

-Added member variables to OrganizationEntity:
     

> Facebook Url

 

> Instragram Url

 

> LinkedIn Url

**-** Added OrganizationPublicDto with public fields of Organization
**-** Added SlideOrganizationPublicDto. Slide with the public fields of Organization.
**-** Modified method to update an Organization because of the new parameters.
**-** Modified method to retrieve a slide with the  corresponding Organization, because of the new parameters.
**-** Modified OrganizationMapper, because of the new parameters.
**-** Modified SlideMapper, because of the new parameters.
**-** Modified OrganizationController to call the modified methods.

### UpdateMethod working:

pr1

### GetSlide methodWorking
pr2

### CODE REVIEW

File: src/main/java/com/alkemy/ong/dto/SlidePublicOrganizationDto.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package com.alkemy.ong.dto;
+
+import lombok.Data;
+
+@Data
+public class SlidePublicOrganizationDto {
+    private String id;
```
Comment (anavduo) [inline src/main/java/com/alkemy/ong/dto/SlidePublicOrganizationDto.java:7]: same comment as above

File: src/main/java/com/alkemy/ong/dto/OrganizationPublicDto.java [hunk 2]
```diff
@@ -0,0 +1,15 @@
+package com.alkemy.ong.dto;
+
+import lombok.Data;
+
+@Data
+public class OrganizationPublicDto {
+    private String id;
```
[the remark this answers is not in the document]
Comment (pablofrankenn) [reply src/main/java/com/alkemy/ong/dto/OrganizationPublicDto.java:9]: Fixed. Added another dto without the Id field.

File: src/main/java/com/alkemy/ong/dto/SlidePublicOrganizationDto.java (hunk 1 above)
Comment (pablofrankenn) [reply src/main/java/com/alkemy/ong/dto/SlidePublicOrganizationDto.java:7]: Ticket explicits that all fields must be included.

File: src/main/java/com/alkemy/ong/dto/OrganizationPublicDto.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (anavduo) [reply src/main/java/com/alkemy/ong/dto/OrganizationPublicDto.java:9]: Great colleague!

**Comment by Lucasfranco1** [comment]
Looks good for me!

File: src/main/java/com/alkemy/ong/dto/SlidePublicOrganizationDto.java (hunk 1 above)
Comment (anavduo) [reply src/main/java/com/alkemy/ong/dto/SlidePublicOrganizationDto.java:7]: Got it!

**Pull request merged**